client/src/index.css

```diff
@@ -1,13 +1,18 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
 }
-
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
+.nav {
```

Comment:
![Screenshot 2024-02-06 at 7 08 40 PM](https://github.com/ylopez25/Tech-dive/assets/79764816/1349ea2e-f20c-4d22-8bfc-c8fe54c1bcfd)

I noticed that when I minimize the window, the navbar becomes scrunched and cut off, if possible can we make a note to have the page adapt depending on browser size, or figure out a way to have it properly resize or have the navbar transition to sandwich menu

I've done this implementation on a previous project

Reply:
I will create a new ticket for the css changes and do a pr for that issue so we can close the navbar issue and have it merged before waiting for these changes